pkgs/tools/audio/yabridge/default.nix

```diff
@@ -0,0 +1,114 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, wine
+, boost
+, libxcb
+}:
+
+let
+  # Derived from subprojects/bitsery.wrap
+  bitsery = rec {
```

Comment:
I would prefer moving this into its own library.

Reply:
I would too, but this project includes patches for bitsery and function2, so I'm not sure if this can be easily separated. I will try though.

Reply:
For context, those patches are for providing Meson build definitions for those libraries since they don't come with them out of the box. I don't know how those situations are normally handled in Nix. For yabridge 3.0/the current master branch yabridge also has to patch a copy of the VST3 SDK during the build process so it can be built with winegcc. That may be a bit more difficult to package (but that's a future concern).

Reply:
@SuperSandro2000 I think it makes sense to leave it as is, because these libraries are cross compiled with `wineg++` and the patches to make them work in `wineg++` are packaged with yabridge.

Reply:
Yeah probably for now.